server/index.js

```diff
@@ -353,6 +353,25 @@ app.delete('/api/saved/remove', authorizationMiddleware, (req, res, next) => {
     })
     .catch(err => next(err));
 });
+
+app.get('/api/saved-posts', authorizationMiddleware, (req, res, next) => {
+  const userId = req.user.user.userId;
```

Comment:
🤔 this is kinda weird looking, like why is there a `req.user.user`? Try to fix your code so that it's just `req.user.userId`